### PR TITLE
Document extra disks for assisted

### PR DESCRIPTION
**Pull request by dtantsur**
It no longer has to be a manual step.

/cc @hardys @flaper87

### CODE REVIEW

File: docs/assisted-deployment.md [hunk 1]
```diff
@@ -8,13 +8,15 @@ the Assisted Service documentation for more information about its architecture a
...
-
-Note that there is work in progress to automate the disk creations and the device name
-configuration. More info in [this issue](https://github.com/openshift-metal3/dev-scripts/issues/1228)
+2 disks need to be created and added to the worker nodes. The disks should be
+of, at least, 10GB each. Set the following variables in your `config.sh`
+before configuring the development environment:
+
+```bash
+export VM_EXTRADISKS=true
+export VM_EXTRADISKS_LIST="vdb vdc"
```
Comment (hardys) [inline docs/assisted-deployment.md:17]: Ref https://github.com/openshift-metal3/dev-scripts/pull/1313/commits/75a147b58ac48c7fb5bb4e5ab1f271c0f8cf3ceb I don't think this list will work - in my testing the extra disk shows up as vda, with the primary OS disk being sda

**Comment by hardys** [comment]
lgtm other than the disk names which I found to be wrong when working on #1313

File: docs/assisted-deployment.md (hunk 1 above)
Comment (dtantsur) [reply docs/assisted-deployment.md:17]: Can confirm :(

File: docs/assisted-deployment.md [hunk 2]
```diff
@@ -8,13 +8,15 @@ the Assisted Service documentation for more information about its architecture a
...
 
-2 disks need to be created and added to the worker nodes. This can be done either using
-`virt-manager` or libvirt directly. The disks should be of, at least, 10GB each and they should be
-attached to the `/dev/sdb` and `/dev/sdc` devices respectively. Please update
-`assisted_deployment.sh` if this is not the case.
-
-Note that there is work in progress to automate the disk creations and the device name
-configuration. More info in [this issue](https://github.com/openshift-metal3/dev-scripts/issues/1228)
+2 disks need to be created and added to the worker nodes. The disks should be
+of, at least, 10GB each. Set the following variables in your `config.sh`
```
Comment (flaper87) [inline docs/assisted-deployment.md:12]:
```suggestion
of, at least, 10GB each. Set the following variables in your `config_$user.sh`
```

**Review by flaper87 — APPROVED** [review]
just a nit, but lgtm

File: docs/assisted-deployment.md (hunk 2 above)
Comment (hardys) [reply docs/assisted-deployment.md:12]: This should be `config_$USER.sh`

**Review by flaper87 — APPROVED** [review]

**Pull request merged**